### ISSUE TITLE
[polyserve] Duplicate CE definition errors in Firefox and Edge when combining paths and bare specifiers in tests

### ISSUE THREAD

**Issue by web-padawan** [Polymer/tools#203]
### Description

Tests for some of the Vaadin components are failing, because of external dependency usage. They pass in Chrome and Safari, but produce the errors like this in Firefox and Edge:

```
"Error: A custom element with name 'vaadin-text-field' has already been defined."
````
Most likely caused by not deduplicating requests originated from modulizer output:
- `<script type="module" src="../../../@vaadin/vaadin-text-field/vaadin-text-field.js"></script>`
- `import '@vaadin/vaadin-text-field/vaadin-text-field.js';`

### Versions & Environment
- Polymer CLI: 1.7.0-pre.13
- node: 8.11.1
- Operating System: Mac OS

### Steps to Reproduce

1. Clone the repo:
```sh
git clone --depth 1 -b p3-preview-bare git://github.com/vaadin/vaadin-list-box.git && cd vaadin-list-box
```

2. Install the modulizer:
```sh
git clone git://github.com/Polymer/polymer-modulizer.git && cd polymer-modulizer && npm link
```

3. Install the tools:
```sh
npm i -g bower yarn magi-cli polymer-cli@next
```

4. Install bower dependencies:
```sh
bower install
```

5. Reset the last commit (containing the workaround):
```
git reset --hard HEAD^1
```

6. Run the conversion:
```sh
magi p3-convert --out . --import-style=name
```

7. Install dependencies:
```sh
yarn installl --flat
```

8. Serve the component:
```sh
polymer serve --npm --module-resolution=node
``` 

9. Open the Firefox and go to http://127.0.0.1:8081/components/@vaadin/vaadin-list-box/test/

### Workaround

We managed to solve this by putting the external HTML imports inside of the different file, then importing that one in test suite, in order to get same paths for HTML and JS. See it here:
- vaadin/vaadin-list-box@0ecfee4 - [Travis green](https://travis-ci.org/vaadin/vaadin-list-box/jobs/370042023)
- vaadin/vaadin-form-layout@6cedbc0 - [Travis green](https://travis-ci.org/vaadin/vaadin-form-layout/jobs/370037928)

### Expected Results (with the workaround):

Tests passing, see the builds above and the screenshot:

![screen shot 2018-04-23 at 14 10 46](https://user-images.githubusercontent.com/10589913/39123872-f6ba90b8-4701-11e8-8400-fbf8e71814cf.png)

### Actual Results (1.7.0-pre.4)

Tests failing, see the builds:
- vaadin/vaadin-list-box@51e5f60 - [Travis red](https://travis-ci.org/vaadin/vaadin-list-box/jobs/368619868)
- vaadin/vaadin-form-layout@a19e2ba - [Travis red](https://travis-ci.org/vaadin/vaadin-form-layout/jobs/368732794)

![screen shot 2018-04-23 at 14 12 11](https://user-images.githubusercontent.com/10589913/39123972-63d3f09a-4702-11e8-80cd-a3696dc23e1d.png)

PS: also looks related to Polymer/polymer-modulizer#408

**Comment by aomarks** [comment]
@web-padawan Just released polymer-cli@polymer-cli@1.7.0-pre.16 with a new AMD loader which may help with this problem. Let me know how it goes!

**Comment by web-padawan** [comment]
I have tried the new version and both broken components now work fine. Thanks!
